src/View/View.php

```diff
@@ -707,12 +711,15 @@ public function render($view = null, $layout = null): string
             $this->dispatchEvent('View.afterRender', [$viewFileName]);
         }
 
-        if ($this->layout && $this->autoLayout) {
+        if ($this->autoLayout) {
```

Comment:
If users are supposed to be able to set `$this->layout = false` in templates shouldn't this also check the layout property?

Reply:
No, one could set `$this->layout = false`  earlier but with this change the alternative to disable layout rendering is use `$this->autoLayout = false`.

Not allowing `View::$layout` to be set to false is the whole point of this PR. Now it showed always contain a string layout name and controlling of layout rendering should be done using `View::$autoLayout` only.